.circleci/config.yml

```diff
@@ -2241,6 +2234,13 @@ workflows:
           context: determined-production
           filters: *release-filters
 
+      - publish-python-package:
```

Comment:
hmm I am not sure just changing the order will help here? I believe this needs `requires:` on either `package-and-push-system-rc` or `package-and-push-system-release`, depending on the filter.

Reply:
This is correct, I'm pretty sure.